tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -314,7 +359,8 @@ Edit methodology
     user clicks button  ${METHODOLOGY_NAME}
     # NOTE: scroll down is here to avoid selenium clicking the
     # set page view text box on the methodology page
-    user scrolls down  400
+    user scrolls down  800
```

Comment:
It's probably better to use `user scrolls to element` in cases like this when you can - I *think* that would be more robust - although I could be wrong. Not a big deal though so you could leave this as it is.